### PR TITLE
Dark Mode: ENS Input

### PR DESCRIPTION
**Pull request by darkwing**
Fixes the ENS input for dark mode.

ENSInput

You can see this component after clicking the SEND button.

### CODE REVIEW

**Review by mcmire — COMMENTED** [review]
Checked your commit and seems pretty straightforward.

File: app/phishing.html [hunk 1]
```diff
@@ -1,67 +1,147 @@
...
+      .content__header h1 i {
+        margin-right: 0.25em;
+      }
+      .content__header img {
+        margin-bottom: 3em;
+        width: 160px;
+      }
+      .content__body {
+        background-color: var(--color-background-alternative);
+        font-size: 12pt;
```
Comment (georgewrmarshall) [inline app/phishing.html:82]: Not sure if it's need or will be covered in our main stylesheet but to be safe add 

```suggestion
        font-size: 12pt;
        color: var(--color-text-default);
```

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Looking good! A couple suggestions

File: ui/components/app/confirm-page-container/confirm-page-container-navigation/index.scss [hunk 2]
```diff
@@ -44,7 +44,7 @@
   &__longtext {
     @include H9;
 
-    color: var(--oslo-gray);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/confirm-page-container/confirm-page-container-navigation/index.scss:47]:
```suggestion
    color: var(--color-text-alternative);
```

File: ui/components/app/connected-accounts-list/index.scss [hunk 3]
```diff
@@ -24,7 +24,7 @@
     @extend %account-status-typography;
 
     display: inline;
-    color: var(--Grey-500);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/connected-accounts-list/index.scss:27]:
```suggestion
    color: var(--color-text-alternative);
```

File: ui/components/app/connected-accounts-permissions/index.scss [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
   display: flex;
   flex-direction: column;
-  color: var(--Grey-500);
+  color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/connected-accounts-permissions/index.scss:6]:
```suggestion
  color: var(--color-text-alternative);
```

File: ui/components/app/connected-status-indicator/index.scss [hunk 5]
```diff
@@ -43,17 +43,17 @@
   }
 
   &__yellow-circle {
-    border-color: #ffd33d;
+    border-color: var(--color-warning-default);
   }
 
   &__grey-circle {
-    border-color: var(--Grey-500);
+    border-color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/connected-status-indicator/index.scss:50]:
```suggestion
    border-color: var(--color-border-default);
```

File: ui/components/app/connected-status-indicator/index.scss [hunk 6]
```diff
@@ -43,17 +43,17 @@
...
   }
 
   &__text {
     @include H8;
 
-    color: var(--Grey-500);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/connected-status-indicator/index.scss:56]:
```suggestion
    color: var(--color-text-alternative);
```

File: ui/components/app/connected-status-indicator/index.scss [hunk 7]
```diff
@@ -8,11 +8,11 @@
   border-radius: 100px;
 
   &:hover {
-    background-color: #f2f3f4;
+    background-color: var(--color-background-alternative);
   }
 
   &:active {
-    background-color: #ededed;
+    background-color: var(--color-background-default);
```
Comment (georgewrmarshall) [inline ui/components/app/connected-status-indicator/index.scss:15]: Can just leave it the same as hover we don't have a color for this. Really it should be using the secondary type Button component

```suggestion
    background-color: var(--color-background-alternative);
```

File: ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss [hunk 8]
```diff
@@ -1,7 +1,7 @@
 .edit-gas-item {
   border-radius: 24px;
-  background: white;
-  color: var(--ui-4);
+  background: var(--color-background-default);
+  color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss:4]:
```suggestion
  color: var(--color-text-alternative);
```

File: ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss [hunk 9]
```diff
@@ -27,7 +27,7 @@
   &__name {
     display: inline-flex;
     align-items: center;
-    color: var(--ui-black);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss:30]:
```suggestion
    color: var(--color-text-default);
```

File: ui/components/app/menu-bar/index.scss [hunk 10]
```diff
@@ -32,6 +33,6 @@
   &__explorer-origin {
     @include H7;
 
-    color: var(--Grey-500);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/menu-bar/index.scss:36]:
```suggestion
    color: var(--color-text-alternative);
```

File: ui/components/app/selected-account/index.scss [hunk 11]
```diff
@@ -43,11 +43,11 @@
     background-color: unset;
 
     &:hover {
-      background-color: var(--Grey-000);
+      background-color: var(--color-background-alternative);
     }
 
     &:active {
-      background-color: #d9d7da;
+      background-color: var(--color-background-default);
```
Comment (georgewrmarshall) [inline ui/components/app/selected-account/index.scss:50]: Just keep it as alternative
```suggestion
      background-color: var(--color-background-alternative);
```

File: ui/components/app/transaction-breakdown/transaction-breakdown-row/index.scss [hunk 12]
```diff
@@ -1,13 +1,13 @@
 .transaction-breakdown-row {
   @include H7;
 
-  color: var(--Grey-500);
+  color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/transaction-breakdown/transaction-breakdown-row/index.scss:4]:
```suggestion
  color: var(--color-text-alternative);
```

File: ui/components/app/transaction-list-item/index.scss [hunk 13]
```diff
@@ -9,15 +9,15 @@
     @include H7;
 
     margin-top: 4px;
-    color: var(--Grey-500);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/transaction-list-item/index.scss:12]:
```suggestion
    color: var(--color-text-alternative);
```

File: ui/components/app/transaction-list-item/index.scss [hunk 14]
```diff
@@ -9,15 +9,15 @@
...
   }
 
   & &--unconfirmed {
-    color: var(--Grey-500);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/transaction-list-item/index.scss:16]:
```suggestion
    color: var(--color-text-alternative);
```

File: ui/components/app/transaction-list-item/index.scss [hunk 15]
```diff
@@ -9,15 +9,15 @@
...
   }
 
   &--unconfirmed &__primary-currency {
-    color: var(--Grey-500);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/app/transaction-list-item/index.scss:20]:
```suggestion
    color: var(--color-text-alternative);
```

File: ui/components/app/transaction-list/index.scss [hunk 16]
```diff
@@ -13,8 +13,8 @@
     @include H6;
 
     flex: 0 0 auto;
-    color: var(--Grey-400);
-    border-bottom: 1px solid var(--Grey-100);
+    color: var(--color-text-default);
```
Comment (georgewrmarshall) [inline ui/components/app/transaction-list/index.scss:16]:
```suggestion
    color: var(--color-text-muted);
```

File: ui/components/ui/nickname-popover/index.scss [hunk 17]
```diff
@@ -47,14 +47,14 @@
     &__constant {
       @include H8;
 
-      color: var(--Grey-500);
+      color: var(--colro-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/ui/nickname-popover/index.scss:50]:
```suggestion
      color: var(--color-text-alternative);
```

File: ui/components/ui/page-container/index.scss [hunk 18]
```diff
@@ -115,7 +115,7 @@
     @include H6;
 
     padding-top: 0.5rem;
-    color: var(--gray);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/ui/page-container/index.scss:118]:
```suggestion
    color: var(--color-text-alternative);
```

File: ui/components/ui/page-container/index.scss [hunk 19]
```diff
@@ -127,7 +127,7 @@
     @include Paragraph;
 
     min-width: 5rem;
-    color: var(--dusty-gray);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/ui/page-container/index.scss:130]:
```suggestion
    color: var(--color-text-alternative);
```

File: ui/components/ui/update-nickname-popover/index.scss [hunk 20]
```diff
@@ -42,7 +42,7 @@
     &__address {
       margin-top: 8px;
       font-size: 13px;
-      color: #bbc0c5;
+      color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/components/ui/update-nickname-popover/index.scss:45]:
```suggestion
      color: var(--color-text-alternative);
```

File: ui/css/itcss/components/network.scss [hunk 21]
```diff
@@ -7,7 +7,7 @@
 }
 
 .network-component.pointer {
-  border: 1px solid var(--Grey-200);
+  border: 1px solid var(--color-border-default);
```
Comment (georgewrmarshall) [inline ui/css/itcss/components/network.scss:10]:
```suggestion
  border: 1px solid var(--color-border-muted);
```

File: ui/css/itcss/components/network.scss [hunk 22]
```diff
@@ -173,7 +173,7 @@
     position: absolute;
     top: 63px;
     right: 10px;
-    border: 1px solid var(--dusty-gray);
+    border: 1px solid var(--color-border-default);
     border-radius: 10px;
   }
 }
```
Comment (georgewrmarshall) [inline ui/css/itcss/components/network.scss:179]: Might be able to remove this file

File: ui/pages/create-account/index.scss [hunk 23]
```diff
@@ -72,7 +72,7 @@
   &__input-label {
     @include Paragraph;
 
-    color: var(--scorpion);
+    color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/pages/create-account/index.scss:75]:
```suggestion
    color: var(--color-text-alternative);
```

File: ui/pages/send/send.scss [hunk 24]
```diff
@@ -96,13 +96,13 @@
     &__group-label {
       @include H8;
 
-      background-color: var(--Grey-000);
-      color: var(--Grey-600);
+      background-color: var(--color-background-alternative);
+      color: var(--color-icon-default);
```
Comment (georgewrmarshall) [inline ui/pages/send/send.scss:100]:
```suggestion
      color: var(--color-text-alternative);
```

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Ah I just realized this PR is based off of the general background PR. Left a lot of updates. @GuillaumeRx

**Comment by darkwing** [comment]
I've addressed all feedback from Guillaume's PR on my `general-background-dark-mode` branch.

**Comment by darkwing** [comment]
Superceded by https://github.com/MetaMask/metamask-extension/pull/13984

**Pull request closed without merging**